src/tools/ontoweave.py

```diff
@@ -180,6 +187,17 @@ def import_from_path(file_path):
     # Late import to avoid useless Biocypher's logs when asking for --help.
     import ontoweaver
 
+    # Validate the input data if asked.
+    if asked.validate_only:
+        logger.setLevel("INFO")
```

Comment:
The user should be in charge of that. For instance, in a script, you would just look at the return code and not the log, hence you should be able to set the log level to nothing.
```suggestion
```